index.js

```diff
@@ -250,8 +250,6 @@ PSPDFKitView.propTypes = {
      * Controls the author name that is set for new annotations.
      * If not set and the user hasn't specified it before the user will be asked and the result will be saved.
      * The value set here will be persisted and the user will not be asked even if this is not set the next time.
```

Comment:
Maybe we should mention the behaviour differences between Android and iOS here? Because as far as is understand the value set here on iOS will NOT be persisted, right?

Reply:
It is persisted. If we set the `annotationAuthorName`, the first time the user adds or edits an annotation, they will be prompted to choose the author name (that is if `shouldAskForAnnotationUsername` is not set to false in `PSPDFConfiguration`). The default value is the one specified by `annotationAuthorName`.

Once the author name has been set, this is persisted app-wide. So the API comment is valid for iOS as well.